### PR TITLE
Add jsp to test matrix application

### PR DESCRIPTION
**Pull request by iNikem**
JSP often use different classloaders and on-the-fly compilation. This demonstrates e.g. #1595

### CODE REVIEW

**Review by mateuszrzeszutek — APPROVED** [review]

File: instrumentation/jaxrs-client/jaxrs-client-2.0/jaxrs-client-2.0-jersey-2.0/javaagent/jaxrs-client-2.0-jersey-2.0-javaagent.gradle [hunk 1]
```diff
@@ -4,7 +4,7 @@ muzzle {
   pass {
     group = "org.glassfish.jersey.core"
     module = "jersey-client"
-    versions = "[2.0,)"
+    versions = "[2.0,3.0.0)"
```
Comment (mateuszrzeszutek) [inline instrumentation/jaxrs-client/jaxrs-client-2.0/jaxrs-client-2.0-jersey-2.0/javaagent/jaxrs-client-2.0-jersey-2.0-javaagent.gradle:7]: Just wondering: why did you add the version limit here?

**Review by mateuszrzeszutek — APPROVED** [review]

File: instrumentation/jaxrs-client/jaxrs-client-2.0/jaxrs-client-2.0-jersey-2.0/javaagent/jaxrs-client-2.0-jersey-2.0-javaagent.gradle (hunk 1 above)
Comment (iNikem) [reply instrumentation/jaxrs-client/jaxrs-client-2.0/jaxrs-client-2.0-jersey-2.0/javaagent/jaxrs-client-2.0-jersey-2.0-javaagent.gradle:7]: Trying to fix sudden muzzle failure :) It seems that there is a new recent version and muzzle failed on that.
